gems/pending/fs/MiqFS/FsProbe.rb

```diff
@@ -14,6 +14,7 @@ module FsProbe
   PROBE_FILES.unshift("Ext4Probe")     if PROBE_FILES.delete("Ext4Probe")
   PROBE_FILES.unshift("Ext3Probe")     if PROBE_FILES.delete("Ext3Probe")
   PROBE_FILES.unshift("NTFSProbe")     if PROBE_FILES.delete("NTFSProbe")
+  PROBE_FILES.unshift("XFSProbe")      if PROBE_FILES.delete("XFSProbe")
```

Comment:
I am under the impression that an early version of the WIP PR, when rebased, somehow ended up deleting this like that was added (or should have been added?) by the XFS PR.  I added it back thinking that it was inadvertent.  If you want it deleted and added under a separate PR let me know.